package.json

```diff
@@ -25,6 +25,7 @@
     "gulp-newer": "^1.3.0",
     "gulp-plumber": "^1.2.1",
     "gulp-postcss": "^8.0.0",
+    "gulp-real-favicon": "^0.3.2",
```

Comment:
@mikeallbutt is this needed?

Reply:
It can be used to generate the favicons so that one doesn't have to keep configuring all the settings, but instead use the now configure settings. The script is only run if desired (when a new favicon is needed).